pkg/kubelet/config/flags.go

```diff
@@ -87,6 +87,7 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 
 	// General settings.
 	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'remote'.")
+	fs.MarkDeprecated("container-runtime", "will be removed in 1.27 as with docker-shim removal only valid value is \"remote\"")
```

Comment:
Should we make this a no-op and remove all the logic behind it?
Also above it's using `'remote'` instead of `\"remote\"`, maybe should keep consistent.